config/sync/views.view.press_release_search.yml

```diff
@@ -408,21 +430,25 @@ display:
     display_plugin: page
     position: 1
     display_options:
+      cache:
+        type: none
```

Comment:
Think we usually use search API cache tags for views, unless there's an overriding reason to set no cache on the output.